zerver/views/scheduled_messages.py

```diff
@@ -20,3 +39,61 @@ def delete_scheduled_messages(
 ) -> HttpResponse:
     delete_scheduled_message(user_profile, scheduled_message_id)
     return json_success(request)
+
+
+@has_request_variables
+def scheduled_messages_backend(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    recipient_type_name: str = REQ("type"),
+    req_to: str = REQ("to"),
+    topic_name: Optional[str] = REQ_topic(),
+    message_content: str = REQ("content"),
+    scheduled_message_id: Optional[int] = REQ(default=None, json_validator=check_int),
+    defer_until: str = REQ("deliver_at"),
+    tz_guess: Optional[str] = REQ("tz_guess"),
```

Comment:
We discussed the ideal semantics in "api design", but I think we should replace these two parameters with a single `send_time` parameter that is required to be UTC, or something like that, and have the client be responsible for sending a precise time?

(I think we needed this mainly for a slash command style interface to ask for something to happen in a few hours, which I'm not sure really makes sense to support at all once we have this first-class feature; "reminders" might be different on that front as a `/remind` slash command seems convenient, but we plan to have a different API anyway).

Reply:
Started CZO #api design conversation for this: https://chat.zulip.org/#narrow/stream/378-api-design/topic/scheduled.20messages.3A.20send_time.20parameter/near/1556114